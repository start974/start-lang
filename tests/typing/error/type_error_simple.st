def test : N2 := 1
