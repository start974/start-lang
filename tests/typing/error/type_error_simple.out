Error[301]: tests/typing/error/type_error_simple.st:1:12
1 | def test : N2 := 1
               ^^ Type name "N2" not found.
